### PR TITLE
Slightly regex changes, now buildPlg runs also on windows

### PR DESCRIPTION
**Pull request by annplaksin**

### CODE REVIEW

File: buildPlg.js [hunk 1]
```diff
@@ -97,8 +97,8 @@ function main(args) {
     console.log('written plugin code to ' + pluginFilename);
   }
 
-  var funcRegex = /function\s*([_a-zA-Z][_a-zA-Z0-9$]*)(\(.*\))\s*\{/;
-  var moduleLineRegex = /^\s*\/\/\$module\(([\w/]+\.mss)\)/;
+  var funcRegex = /function\s*([_a-zA-Z][_a-zA-Z0-9$]*)\s*(\(.*\))\s*\{/;
```
Comment (zolaemil) [inline buildPlg.js]: 👍

**Comment by annplaksin** [comment]
The ^ and $ just not work with Windows because it doesn't support multiline regex. As I remember, the regex is processed by cmd.exe. I'm not familiar in using regex on Windows and this was quite a quick fix to make it work. 
Maybe there is a better way to solve this problem, but I was not able to find it yet.

**Comment by davethehat** [comment]
Hi Anna,

I was going to take a look at this later - just to verify that it doesn't
break the Mac.version.

Best

David

On 9 Jan 2017 2:04 pm, "Anna Plaksin" <notifications@github.com> wrote:

The ^ and $ just not work with Windows because it doesn't support multiline
regex. As I remember, the regex is processed by cmd.exe. I'm not familiar
in using regex on Windows and this was quite a quick fix to make it work.
Maybe there is a better way to solve this problem, but I was not able to
find it yet.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tido/plgToMSS/pull/4#issuecomment-271291975>, or mute
the thread

**Comment by annplaksin** [comment]
Hi David,

sounds like a good idea. I'm obviously not a Mac user and cannot test it by myself. I really hope it doesn't break it on Mac. That would make this issue more complicated.

Regards,
Anna

**Comment by th-we** [comment]
Just wanted to chime in: I'm running this on Linux (with Sibelius in a VM), and Anna's version worked for me as well, while the original one didn't. I have not further investigated why, just wanted to confirm that this pull request is actually sensible.

**Comment by davethehat** [comment]
Thanks Thomas. I've been snowed under these last few weeks, but I will take
a look at this on my various machines soon.

Thanks for your patience,

David

**Comment by davethehat** [comment]
Checked and merged. Many thanks Anna, and once again apologies for taking a bit of time to check and merge this.

Best wishes
David

**Pull request merged**